### ISSUE TITLE
Please implement reset buttons for gestures-controlled volume and brightness.

### ISSUE THREAD

**Issue by getphobo** [TeamNewPipe/NewPipe#6501]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want
Using the fullscreen gestures, NewPipe changes the volume and/or brightness and remembers their settings for fullscreen mode. A button to reset them to the device's default values would be nice. I use a phone with automatic brightness control (Google Pixel 2 running Android 11), so having an auto-reset button would be even more helpful.

#### Is your feature request related to a problem? Please describe it
I sometimes just want to reset the fullscreen volume or brightness to the device's default setting.

#### How will you/everyone benefit from this feature?
It's as helpful as a reset function can get. It might help people who have difficulties using the fullscreen gestures.

**Comment by SameenAhnaf** [comment]
![IMG_20210617_164151](https://user-images.githubusercontent.com/79245641/122382233-88f39280-cf8b-11eb-8ba3-f8f85d173db9.jpg)

I think, volume and brightness sliders need to be redesigned a little. At this moment, they are unnecessarily huge. Your desired buttons could be placed above (as in the screenshot) or below the sliders. 

Note that, these sliders should be included in video interface as well as it is quite hard to change these settings there. Also, only related slider on the opposite side while using any gesture.

Reset: Change brightness according to device
Extra: Enables users to boost volume. Useful when any video has too low volume.

**Comment by mhmdanas** [comment]
@SameenAhnaf I don't think that volume and brightness sliders are implemented in the app currently. If you mean the system ones, these differ significantly between OEMs. Can you clarify?

**Comment by SameenAhnaf** [comment]
@mhmdanas tbh, I dunno how to address the round icons in Newpipe app. I thought, both "Big round icons" (as of Newpipe app) and "Vertical sliders" (as of my MX Player app screenshot, not system) are called sliders.
